integration_test/Dialect/Handshake/tuple_packing/tuple_packing.mlir

```diff
@@ -1,4 +1,4 @@
-// REQUIRES: iverilog,cocotb
+// REQUIRES: iverilog,cocotb,python
```

Comment:
Shouldn't cocotb imply python?

Reply:
`cocotb` is checked by importing it in the `lit` config, which did previously not imply that python was checked.  Now that it's ensured that a python3 executable is detected and remembered, this should be the case.

I can drop this, I guess.